src/components/MessagesList/MessagesList.vue

```diff
@@ -588,11 +588,24 @@ export default {
 		 */
 		smoothScrollToBottom() {
 			this.$nextTick(function() {
-				this.scroller.scrollTo({
-					top: this.scroller.scrollHeight,
-					behavior: 'smooth',
-					 })
-				this.setChatScrolledToBottom(true)
```

Comment:
I think this is actually what the biggest problem was. It removed the scroll-to-bottom button, although it wasn't scrolled to the bottom